x/wasm/client/cli/genesis_msg.go

```diff
@@ -333,28 +317,22 @@ func GetAllContracts(state *types.GenesisState) ([]ContractMeta, error) {
 		}
 	}
 	// add inflight
+	seq := contractSeqValue(state)
 	for _, m := range state.GenMsgs {
 		if msg := m.GetInstantiateContract(); msg != nil {
```

Comment:
Do we need to add a case here for the new `InstantiateContract2` in `GenMsgs`?

Reply:
I commented on the `GenMsgs` proto that this is not supported. It is a very complex part of the system that we should remove instead https://github.com/CosmWasm/wasmd/issues/987